peril.settings.json

```diff
@@ -10,7 +10,7 @@
     // Jira integration
     "pull_request": ["org/allPRs.ts", "org/jira/pr.ts", "org/mobilePRsForbidForks.ts"],
     "pull_request.closed": ["org/closedPRs.ts", "org/jira/pr.ts"],
-    "pull_request.opened": "org/addPatchLabel.ts",
+    "pull_request.opened": ["org/openedPR.ts", "org/addPatchLabel.ts"],
     // The RFC process
```

Comment:
I'm surprised this works. I thought you couldn't have comments in JSON files... :thinking: 

Maybe Peril does some pre-processing of the JSON file before parsing it?

Reply:
yeah same, as far as I'm concerned it's just magic

Reply:
That's right. Comments don't normally work in JSON files because JavaScript's `JSON.parse()` can't handle them. There're a few packages that existing to extend the format to support comments. npm could, if it wanted to, use the same approach that Peril does for `package.json`. However, they've stated that they won't because they do no want to break compatibility with `JSON.parse()`. Orta must have decided the trade-offs were worth it for Peril 😄